doc/users-accounts.md

```diff
@@ -0,0 +1,82 @@
+# User accounts
+
+## Introduction
+
+Our users sign in to access the applicaiton.
+
+The authentication is handled by the DfE Active Directory via Omniatuth.
+
+We store registered users in a local table, and authorise them to perform tasks
```

Comment:
Could "local table" just be "database table" or something. I think "local table" might have meaning outside of what this is saying